starter_files/lib/deck.rb

```diff
@@ -0,0 +1,33 @@
+require_relative 'card'
+
+class Deck
+  attr_accessor :deck
+  def initialize
+    @deck = []
+    ranks = [:A, 2, 3, 4, 5, 6, 7, 8, 9, 10, :J, :Q, :K]
+    ranks.each do |rank|
+      @deck << Card.new(rank, :clubs)
+    end
+    ranks.each do |rank|
+      @deck << Card.new(rank, :diamonds)
+    end
+    ranks.each do |rank|
+      @deck << Card.new(rank, :hearts)
+    end
+    ranks.each do |rank|
+      @deck << Card.new(rank, :spades)
+    end
+      end
```

Comment:
You've got some repeated code here -- see if you can think of a way to refactor this.